### DISCUSSION TITLE
Add text alignment for items in Item Lists

### DISCUSSION THREAD

**Discussion by drakonbane** [godotengine/godot-proposals#7007] [GUI]
As title says. As of now, the text inside Item List Items can only be left aligned. The text can be pseudo-aligned with Fixed Column Width but it is not very convenient. The ability to align the text inside Item List Items would open up more UI options.

**Comment by AThousandShips** [comment]
I'll take a look at this actually, I have some ideas

**Reply by LordMcMutton** [reply]
I don't know how far you got with this, but the behavior already exists with the Layout Direction option- setting it to Right-to-Left and then reworking the Container settings results in right-aligned text. 

Perhaps this can be utilized for full alignment control?

**Reply by AThousandShips** [reply]
Never got anywhere with this actually, so if anyone else is interested that could be an idea too
